src/__tests__/core/web_api/__snapshots__/helper.test.js.snap

```diff
@@ -1,15 +1,14 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`normalizeError maps COA not enabled error 1`] = `
+exports[`normalizeError should map access_denied error to invalid_user_password when error.code === access_denied 1`] = `
 Object {
-  "code": "cross_origin_login_not_enabled",
-  "description": "Cross Origin Authencation is not enabled. Please refer to this document to enable it: https://link-to-doc.com",
-  "error": "cross_origin_login_not_enabled",
-  "logToConsole": true,
+  "code": "invalid_user_password",
+  "description": "foobar",
+  "error": "invalid_user_password",
```

Comment:
What's the reason behind having code and error with the same values? Shouldn't you keep 1, `code` probably?

Reply:
I don't know why it is like this. Can't change this now though.